src/styles/pages/_live.scss

```diff
@@ -207,6 +230,36 @@ table.w-event-schedule {
     // needlessly pads out each row.
     margin-bottom: 0;
   }
+
+  .w-event-schedule__open {
+    pointer-events: auto;
+    display: inline-block;
+    color: inherit;
+
+    span {
+      z-index: 1;
+      position: relative;
+    }
+
+    // This creates a large clickable area around the link itself, but which
+    // doesn't extend into the speakers area on the left.
+    &::before {
+      content: '';
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      display: block;
+      position: absolute;
+      z-index: 0;
+    }
+
+    // When the link is hovered, highlight the whole row, even though the
```

Comment:
I think this is creating an effect where once you hover the author, the row hover goes away which creates a kind of flicker that we probably don't want.

Our blog post card elements retain their hover state even when you're hovering over sub items. I'm going to see if I can figure out how to do similar css with these rows and offer that as a suggestion.

Reply:
I pushed a CSS tweak